From f433aaf8c8f82f0aeaedee664f08bc6fcad47b0d Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Wed, 12 Jan 2022 23:45:53 +0100
Subject: [PATCH] Remove superfluous arguments to i18n.merge_file

It fixes the build with meson 0.61.0. The positional argument was
ignored in the past, became a warning in meson 0.60 and a hard error
starting with 0.61.0.
---
 accounts-service/meson.build   | 2 +-
 malcontent-control/meson.build | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/accounts-service/meson.build b/accounts-service/meson.build
index 6942d2b..198692c 100644
--- a/accounts-service/meson.build
+++ b/accounts-service/meson.build
@@ -1,4 +1,4 @@
-i18n.merge_file('com.endlessm.ParentalControls.policy',
+i18n.merge_file(
   input: 'com.endlessm.ParentalControls.policy.in',
   output: 'com.endlessm.ParentalControls.policy',
   po_dir: po_dir,
diff --git a/malcontent-control/meson.build b/malcontent-control/meson.build
index 8263306..d4ef3c1 100644
--- a/malcontent-control/meson.build
+++ b/malcontent-control/meson.build
@@ -36,7 +36,7 @@ malcontent_control = executable('malcontent-control',
   install: true,
 )
 
-desktop_file = i18n.merge_file('desktop-file',
+desktop_file = i18n.merge_file(
   type: 'desktop',
   input: '@0@.desktop.in'.format(application_id),
   output: '@0@.desktop'.format(application_id),
@@ -57,7 +57,7 @@ if desktop_file_validate.found()
   )
 endif
 
-appdata_file = i18n.merge_file('appdata-file',
+appdata_file = i18n.merge_file(
   input: '@0@.appdata.xml.in'.format(application_id),
   output: '@0@.appdata.xml'.format(application_id),
   po_dir: join_paths(meson.current_source_dir(), '..', 'po'),
@@ -94,7 +94,7 @@ if xmllint.found()
   )
 endif
 
-policy_file = i18n.merge_file('policy-file',
+policy_file = i18n.merge_file(
   input: '@0@.policy.in'.format(application_id),
   output: '@0@.policy'.format(application_id),
   po_dir: join_paths(meson.current_source_dir(), '..', 'po'),
-- 
GitLab

